spec/integration/base_spec.rb

```diff
@@ -135,6 +135,25 @@ class Book < ActiveFedora::Base
     end
   end
   
+  describe "#apply_schema" do
+    before do
+      class ExampleSchema < ActiveTriples::Schema
+        property :title, predicate: RDF::DC.title
+      end
+      class ExampleBase < ActiveFedora::Base
+        apply_schema ExampleSchema
+      end
+    end
+    after do
+      Object.send(:remove_const, :ExampleSchema)
+      Object.send(:remove_const, :ExampleBase)
+    end
+    let(:obj) { ExampleBase.new }
+    it "should configure properties and solrize them" do
+      obj.title = ["Test"]
+      expect(obj.to_solr[ActiveFedora::SolrQueryBuilder.solr_name("title", :symbol)]).to eq ["Test"]
```

Comment:
AT Schemas inherently don't have a concept of indexing hints. The proposal is that the default strategy always indexes somehow, because that's the primary access mechanism. If a more granular strategy is desired I could imagine a strategy that takes a property -> index map and applies it conditionally.